### PR TITLE
Correct attribute name to be the same used by Android

### PR DESCRIPTION
**Pull request by Mrbenoit624**
Related with #953

**Changes**

Find a way to keep all information about AndroidManifest, I didnt find a file with issue mention in #953 there so if a sample is available it could be very nice, but normally it should not affect the number of information provided by the AndroidManifest.

Please don't merge it without review and test with the file issue. I let @danielgf3 answer about that.

### CODE REVIEW

**Comment by danielgf3** [comment]
You can find examples in the PR description. https://github.com/androguard/androguard/issues/953#issue-1957025765

**Comment by Mrbenoit624** [comment]
I see the hash and the report on VT but I don't where I can download files

**Comment by Mrbenoit624** [comment]
Hi @danielgf3 , I don’t know if you see my message. I can’t get the files from VT could you send me the files.

thank you

**Comment by danielgf3** [comment]
> Hi @danielgf3 , I don’t know if you see my message. I can’t get the files from VT could you send me the files.
> 
> thank you

Where could I send you the files?

**Comment by Mrbenoit624** [comment]
I send you an email with details to send me files @danielgf3

**Comment by Mrbenoit624** [comment]
Thank you @danielgf3 I test your two samples and there are not your errors anymore.

I did two type of tests:

- androguard axml <sample>
- a custom script just there:

```python
from androguard.misc import AnalyzeAPK
import sys

apk, dex, vm = AnalyzeAPK(sys.argv[1])
print(apk.package)
manifest = apk.axml.get("AndroidManifest.xml")
print(manifest.root.get("platformBuildVersionCode"))
print(manifest.root.get("platformBuildVersionName"))
```

The diffing for all tests is the same than the main branch and only the attribute problem is now fix.

It's ready for a merge thank you

**Comment by Mrbenoit624** [comment]
Hi here, do you know when the next release will be published ?

**Comment by erev0s** [comment]
hi @Mrbenoit624, towards the end of this month.

File: androguard/core/axml/__init__.py [hunk 1]
```diff
@@ -821,14 +822,18 @@ def getAttributeName(self, index):
 
         res = self.sb[name]
         # If the result is a (null) string, we need to look it up.
-        if not res or res == ":":
+        if name <= len(self.m_resourceIDs):
```
Comment (obfusk) [inline androguard/core/axml/__init__.py:825]: @erev0s shouldn't this be `<` instead of `<=`?

Comment (erev0s) [reply androguard/core/axml/__init__.py:825]: nice catch!

**Pull request merged**